changelog/fix-continue-course-block-redirect-using-plain-permalinks

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fixed
+
+Fix continue course button not working when using plain permalinks
```

Comment:
As it's under the type 'fixed' already, should we remove the word 'Fix' from here? Just to take care of the redundancy thing we discussed.

Reply:
Thanks for the reminder. Fixed in d11eed237d90b936918d0a0a0da987bbff6ffa38.